src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -737,7 +823,7 @@ export default {
 		this.toggleFocusTrap()
 	},
 
-	beforeDestroy() {
+	beforeUnmount() {
```

Comment:
Have you checked this is working? I'm 99.99% sure new names are not backported to Vue 2.7...